runtime/tools/spec-builder/test/actual-raw-files/customSpecRaw-code.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I think that `actual-raw-files` should be git ignored. There is no point in committing them, similar to `node_modules` and `build` files. There are simply outputs of some process